src/config.ts

```diff
@@ -0,0 +1,36 @@
+import { Container } from 'aurelia-dependency-injection';
+import { Validator } from './validator';
+import { StandardValidator } from './implementation/standard-validator';
+import { validateTrigger } from './validate-trigger';
+
+/**
+ * Aurelia Validation Configuration API
+ */
+export class AureliaValidationConfiguration {

```

Comment:
This name doesn't feel like it can reflect the global usage intention. Maybe just replace Aurelia with global, though for naming, let's pull in @EisenbergEffect and @fkleuver and @jdanyow

Reply:
Alrigt, I'll wait to hear what @EisenbergEffect, @fkleuver and @jdanyow say.

Reply:
Another ping for @EisenbergEffect, @fkleuver and @jdanyow. Could one of you please lend your insight on an appropriate name for this class?

Reply:
@bigopon 's idea, `GlobalValidationConfiguration`, sounds good to me actually

Reply:
`GlobalValidationConfiguration` it is.